cmd/deployIntegrationArtifact_test.go

```diff
@@ -29,8 +28,8 @@ func TestRunDeployIntegrationArtifact(t *testing.T) {
 		config := deployIntegrationArtifactOptions{
 			Host:                   "https://demo",
 			OAuthTokenProviderURL:  "https://demo/oauth/token",
-			Username:               "sb-8255b149-c3e6-417e-ad27-21fa5a3349dd!b15187|it!b11463",
-			Password:               "911e13b3-312f-4644-9607-7c21974cb0d6$a1sii7gpT3h_242UKSJLbKnV8wKyeQ6qCsQTxEmvDfE=",
+			Username:               "sb-8f9-c3e6-417e-ad27-21fa5a3349dd!b15187|it!b11463",
+			Password:               "9f43-312f-4644-9607-7c21974cb01sii7gpT3h_242UKSJLbKnV8wKyeQ6qCsQTxEmvDfE=",
```

Comment:
Why was the credentials changed. Do you run against some real backend.

If yes: this is a no-go in the unit tests. Having real backends in unit tests would be like a kill-switch for the unit tests (... in case the backend is not available for any reason), or if the tests are executed locally without connectivity.

In case we go against real backends here, please replace by mocks.